pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,317 @@
+    Difference between simulated and real quantum computer will be that:
+    - There is no noise on a simulated quantum computer (so results are better)
+    - Real quantum computer are quicker than simulator
+    - Real quantum computer tasks are assigned to a queue
+      before being executed on a back-end
+
+    WARNING: At the moment this implementation only supports binary
+    classification (eg. Target vs Non-Target experiment)
+
+    Parameters
+    ----------
+    target : int
+        Label of the target symbol
+    qAccountToken : string (default:None)
+        If quantum==True and qAccountToken provided,
+        the classification task will be running on a IBM quantum backend
+    processVector : lambda vector: processedVector (default)
+        Additional processing on the input vectors. eg: downsampling
+    verbose : bool (default:True)
+        If true will output all intermediate results and logs
+    quantum : Bool (default:True)
+        - If true will run on local or remote backend
+        (depending on qAccountToken value).
+        - If false, will perform classical computing instead
+    **parameters : dictionnary
+        This is used by  SKLearn with get_params and set_params method
+        in order to create a deepcopy of an instance
+
+    Attributes
+    ----------
+    classes_ : list
+        list of classes.
+    verbose : see above
+    processVector : see above
+    qAccountToken : see above
+    target : see above
+    quantum : see above
+    test_input : Dictionnary
+        Contains vectorized test set for target and non-target classes
+    training_input : Dictionnary
+        Contains vectorized training set for target and non-target classes
+    provider : IBMQ Provider
+        This service provide a remote quantum computer backend
+    backend : Quantum computer or simulator
+    feature_dim : int
+        Size of the vectorized matrix which is passed to quantum classifier
+    new_feature_dim : int
+        Feature dimension after proccessed by `processVector` lambda
+    prev_fit_params : Dictionnary of data and labels
+        Keep in memory data and labels passed to fit method.
+        This is used for self-calibration.
+    feature_map: ZZFeatureMap
+        Transform data into quantum space
+    quantum_instance: QuantumInstance (Object)
+        Backend with specific parameters (number of shots, etc.)
+
+    See Also
+    --------
+    QuanticSVM
+    QuanticVQC
+
+    """
+
+    def __init__(self, target, qAccountToken=None, quantum=True,
+                 processVector=lambda v: v, verbose=True, **parameters):
+        self.verbose = verbose
+        self.log("Initializing Quantum Classifier")
+        self.test_input = {}
+        self.set_params(**parameters)
+        self.processVector = processVector
+        self.qAccountToken = qAccountToken
+        self.training_input = {}
+        self.target = target
+        self.quantum = quantum
+        if quantum:
```

Comment:
Have a look to this excellent comment:
https://github.com/pyRiemann/pyRiemann/pull/122#discussion_r648021748

Reply:
Thank you for sharing the comment, you are right.